### ISSUE TITLE
Regression from closed source: Many 0-length frames in decoding

### ISSUE THREAD

**Issue by colintheshots** [INDExOS/media-for-mobile#23]
The open source media for mobile library appears to regress from the old JAR we had. On transcoding using my Nexus 6, we're finding many video frames are now decoded as 0-length. The final, transcoded video then has many dropped frames. Reverting to the JARs of version 1.2.2675 fixes this regression for us.

We cannot easily pinpoint the source of this regression because the open source library was committed without history.

**Comment by asuhov** [comment]
Thank you for reporting this issue, we will take a look at it.
Could you clarify the origin of the input video stream? Was it recorded using phone's camera or copied from other source? If you could share the stream, it would help.

**Comment by colintheshots** [comment]
Yes, we simply imported the video from an mp4 movie file. I'll attach it. 
[VidkuHQsnowstorm.mp4.zip](https://github.com/INDExOS/media-for-mobile/files/221473/VidkuHQsnowstorm.mp4.zip)

**Comment by colintheshots** [comment]
For parameters, we were changing the video from 1080x1920 to 720x1280. Rotation 0.
